modules/client/src/test/java/org/apache/ignite/client/fakes/FakeInternalTable.java

```diff
@@ -492,4 +493,10 @@ private void onDataAccess(String operation, Object arg) {
     public @Nullable PendingComparableValuesTracker<Long, Void> getPartitionStorageIndexTracker(int partitionId) {
         return null;
     }
+
+    @Override
+    public ScheduledExecutorService streamerFlushExecutor() {
+        //noinspection DataFlowIssue
+        return null;
```

Comment:
Would it make sense to throw an `UnsupportedOperationException` instead of returning a `null` (that might make something else explode way later)?

Reply:
Done.